packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -149,10 +155,15 @@ class AppBar extends StatelessWidget {
 
   double get _toolBarHeight => kToolBarHeight;
 
-  /// The height of this app bar when fully expanded.
+  /// By default: the height of the toolbar and the tabbar (if any).
```

Comment:
I just meant that if you don't specify this parameter, then by default it's value is...